### ISSUE TITLE
add range reservation and explain why zeroth and last codepoint matter

### ISSUE THREAD

**Issue by job** [large-bgp-communities/large-bgp-communities#9]
inspiration can be taken from https://tools.ietf.org/html/draft-haas-code-point-reservation-bcp-02

**Comment by job** [comment]
```
            0:0:0          -          0:4294967295:4294967295
            65535:0:0      -      65535:4294967295:4294967295
            4294967295:0:0 - 4294967295:4294967295:4294967295
```

**Comment by job** [comment]
fixed in 171ccb5ff1fa65f03aeaeb3605048f2cdbd28eb6
